src/Learning/KWData/KWCYac.cpp

```diff
@@ -923,253 +923,253 @@ static void yydestruct(const char* yymsg, yysymbol_kind_t yykind, YYSTYPE* yyval
 	switch (yykind)
 	{
 	case YYSYMBOL_BASICIDENTIFIER: /* BASICIDENTIFIER  */
-#line 134 "D:/Users/miib6422/Documents/boullema/DevGit/khiops/src/Learning/KWData/KWCYac.yac"
+#line 134 "C:/Applications/boullema/DevGit/khiops/src/Learning/KWData/KWCYac.yac"
```

Comment:
Ce problème se passe pour flex et bison

En cherchant dans les options de bison, j'ai découvert une option avancée:
_Directive: %no-lines
    Don’t generate any #line preprocessor commands in the parser implementation file. Ordinarily Bison writes these commands in the parser implementation file so that the C compiler and debuggers will associate errors and object code with your source file (the grammar file). This directive causes them to associate errors with the parser implementation file, treating it as an independent source file in its own right._ 

Et pour flex, il y a une directive équivalente _%noline_

Ces directives rendent un vrai service pour le développeur/mainteneur des langages kdic et json, en permettant de piloter correctement le debugguer vers les fichiers sources de la grammaire (.lex et .yac).

On pourrait envisager de les enlever à la demande, mais il faudrait alors:
- étendre l'option "BUILD_LEX_YACC" du fichier CMakePresets.json, avec la possibilité de générer ou non les _#line_
- faire attention lors du processus de maintenance des langages à réactiver les directives #noline et re-générer les grammaires avant le commit

Le rapport coût/bénéfice me semble faible.

Reply:
Je ne sais pas comment paramétrer flex et bison pour qu'ils gèrent les paths en relatif plutôt qu'absolu.
Les outils bison et flex ont été upgradés entre dev-v10 et dev, ainsi que les commandes de lancement de ces outils.

Sur mon poste local, je me placait dans le répertoire des sources et je lancais bison directement (win_bison -o%1 %2).
Il est probable qu'avec le passage sur github et le paramétrage de makefile, cela soit fait différemment.
Je pense qu'il faudrait regarder de ce côté.

Reply:
The change to absolute paths has been made in commit 187a8854ec38f95d64e62baff5fccd223009374d.

Reply:
OK, I will add issue to add `%noline` (flex) and `%no-lines` (bison) to the CMake presets.

Reply:
Added issue https://github.com/KhiopsML/khiops/issues/502.